app/services/idv/steps/inherited_proofing/verify_wait_step_show.rb

```diff
@@ -0,0 +1,41 @@
+module Idv
+  module Steps
+    module InheritedProofing
+      class VerifyWaitStepShow < VerifyBaseStep
+        include UserPiiManagable
+        delegate :controller, :idv_session, to: :@flow
+
+        STEP_INDICATOR_STEP = :getting_started
+
+        def call
+          poll_with_meta_refresh(IdentityConfig.store.poll_rate_for_verify_in_seconds)
+
+          process_async_state(async_state)
+        end
+
+        private
+
+        def process_async_state(current_async_state)
+          if current_async_state.none?
+            if inherited_proofing_form_response.success?
+              mark_step_complete(:verify_wait)
```

Comment:
Is the idea here that if there is not async state we submit the form and wait for the results serially? It may be better to render an error and send the user back with that as a flash. That way we don't end up with blocking requests here, especially since we know these may take a long time.

Reply:
Definitely trying to do async here. (But not doing it well! 😅 )